vendors/espressif/boards/esp32/ports/ble/nimble/iot_ble_hal_gap.c

```diff
@@ -43,6 +43,11 @@
 
 BTBleAdapterCallbacks_t xBTBleAdapterCallbacks;
 static struct ble_gap_adv_params xAdv_params;
+
+#define IOT_BLE_ADVERTISING_DURATION_UNITS    ( 10 )
```

Comment:
10 is a quantity rather than a unit of measure, suggest renaming to `IOT_BLE_ADVERTISING_DURATION_MS` since I think the units are milliseconds based on the comment on line 678 below.